metabase/metabase
# Adding is-true and is-false operators

Closes https://github.com/metabase/metabase/issues/14939

### Description
Adds `is true` and `is false` operators when a boolean column is selected for conditional formatting in a table. Note that this will not work on a MySQL table as currently those columns come back as `type/Integer`. We also do our best to keep boolean columns from identifying as string columns, as their semantic type of `category` means they can sometimes be interpreted as string type columns.

Also adjusted some text when multiple columns are selected

### How to verify
The `Accounts` table in the sample database has boolean columns that can be used for testing. Try adding rules for `is true` and `is false` on that table. Note that only like columns should be selectable when creating the rules (can't select a number and a string column for example). 

### Demo
![chrome_5lhEDm7dDe](https://github.com/metabase/metabase/assets/1328979/18e12221-20a4-4513-b8ca-f74557c2afef)


### Checklist
